Assets/Scripts/Quest/MoraleDisplay.cs

```diff
@@ -9,7 +10,7 @@ public class MoraleDisplay {
     public MoraleDisplay()
     {
         Game game = Game.Get();
-        md = new DialogBox(new Vector2(0.75f, 0.5f), new Vector2(3, 3), game.quest.morale.ToString(), Color.red);
+        md = new DialogBox(new Vector2(0.75f, 0.5f), new Vector2(3, 3), new StringKey(game.quest.morale.ToString(),false), Color.red);
```

Comment:
Yes. therer are lot of no needing translation strings. I will fix it by creating a DialogBox and TextBox constructor including numbers but I for the rest, I think all texts in screen should be StringKeys